src/style.css

```diff
@@ -0,0 +1,156 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* #region container */
+.item-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}

```

Comment:
Also add padding: 32px 16px 16px

Reply:
Be more attentive to my comments

![image](https://user-images.githubusercontent.com/63306833/125911534-144f8928-63ba-4841-9701-151db1820221.png)